scte35/state_test.go

```diff
@@ -705,3 +738,207 @@ func TestOutInInOutIn36_37_10_37_37(t *testing.T) {
 		t.Errorf("There should have been 1 open signal, but we saw (%d) open signals instead.", len(state.Open()))
 	}
 }
+
+// Tests closing logic of 0x11 with 0x10, 0x20 ,0x30
```

Comment:
I combined multiple open signals here to keep unit tests short. I had to write separate tests for 0x22 and 0x34,0x36,0x40 and 0x44 as the closing logic between 0x11 and these segmentation types intersect making it hard to test them in 1 test.